packages/ui5-config/src/types.ts

```diff
@@ -15,7 +15,6 @@ export interface ProxyUIConfig {
     [key: string]: unknown | undefined;
     path?: string[];
     url?: string;
-    version?: string;
```

Comment:
The `version` is still a valid property. It just doesn't need to be set by default. I would keep it in the type's definition.

Reply:
Isnt it redundant as its not referenced anymore?